templates/mod.html

```diff
@@ -463,6 +478,32 @@ <h4 class="modal-title" id="myModalLabel">Delete Version</h4>
     </div>
 </div>
 {% endif %}
+{% if admin %}
+<div class="modal fade" id="confirm-lock" tabindex="-1" role="dialog" aria-labelledby="confirm-lock" aria-hidden="true">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+                <h4 class="modal-title" id="myModalLabel">Lock Mod</h4>
+            </div>
+            <form action="{{ url_for("mods.lock", mod_id=mod.id) }}" method="POST">
+                <div class="modal-body">
+                    <p>
+                    Are you really, really, really sure you want to lock this mod?
+                    It will be unpublished and not visible for other users besides the owner anymore.</p>
```

Comment:
"It will be unpublished and hidden for other users besides the owner" or "It will be unpublished and hidden for other users besides the author"